sdk/java/dagger-java-annotation-processor/src/main/java/io/dagger/annotation/processor/DaggerModuleAnnotationProcessor.java

```diff
@@ -69,9 +71,27 @@ ModuleInfo generateModuleInfo(Set<? extends TypeElement> annotations, RoundEnvir
     Set<ObjectInfo> annotatedObjects = new HashSet<>();
     boolean hasModuleAnnotation = false;
 
+    Map<String, EnumInfo> enumInfos = new HashMap<>();
+
     for (TypeElement annotation : annotations) {
       for (Element element : roundEnv.getElementsAnnotatedWith(annotation)) {
-        if (element.getKind() == ElementKind.PACKAGE) {
+        if (element.getKind() == ElementKind.ENUM && element.getAnnotation(Enum.class) != null) {
```

Comment:
Nit: A switch case may more appropriate, so if at some point you add other annotations, you don't have to change the whole block, just add a new statement.

 *(It's an early "optimization" so feel free to ignore it if you do not find that useful)*

Reply:
done 👍